src/components/venue-list/venue/venue-component.scss

```diff
@@ -0,0 +1,30 @@
+.venue {
+  display: flex;
+  padding: 20px 24px;
+  margin: 0 -24px;
+  box-shadow: inset 0px -1px 0px rgba(255, 255, 255, 0.24);
```

Comment:
nitpicky: Insert an empty line between bracketed areas in scss files.  It helps declutter things.

Reply:
Nice! Just added spaces 👍